src/einsteinpy/symbolic/helpers.py

```diff
@@ -187,3 +188,106 @@ def inv(self):
         tmp_array = np.reciprocal(tmp_array)
         self._inv = ImmutableDenseNDimArray(tmp_array)
         return self._inv
+
+
+def get_config_combinations(rank: int) -> list:
```

Comment:
```suggestion
def _get_config_combinations(rank: int) -> list:
```
This is not a user-facing function. Let's make this hidden